docker-compose.yml

```diff
@@ -0,0 +1,90 @@
+version: '3'
+services:
+    factomd:
+        build:
+            context: https://github.com/FactomProject/factomd.git
+            dockerfile: Dockerfile.alpine
+        volumes:
+            # Volume where the blockchain is located.
+            - ./volumes/factomd:/root/.factom/m2
+            # Volume that holds the factomd.conf file.
+            - ./config/factomd.conf:/root/.factom/m2/factomd.conf:ro
+        expose:
+            # Factomd API
+            - 8088
+        ports:
+            # Control panel
+            - 8090:8090
+            # P2P
+            - 8108:8108
+        restart: unless-stopped
+        container_name: factomd
+        logging: &Logging
+            options:
+                max-size: "500k"
+                max-file: "10"
+    walletd:
+        build:
+          context: https://github.com/FactomProject/factom-walletd.git
+          dockerfile: Dockerfile.alpine
+        volumes:
+            # Volume where wallet information is stored
+            - ./volumes/walletd:/root/.factom/wallet
+            # Volume that holds the factomd.conf file
+            - ./config/factomd.conf:/root/.factom/m2/factomd.conf:ro
+        expose:
+            # Wallet API
+            - 8089
+        depends_on:
+            - factomd
+        restart: unless-stopped
+        container_name: factom_walletd
+        logging: *Logging
+    pegnet:
```

Comment:
Should the pegnet and pegnet-coordinator services have a `depends-on` field for factomd and walletd?